File: loops/fizzbuzz.py
# Elizabeth Gutierrez, FizzBuzz in Python
x = 0
nums = int(input("Choose Number: "))
while x <= nums:
    if x % 3 == 0 and x % 5 == 0:
        print("fizzbuzz")
    elif x % 3 == 0:
        print("fizz")
    elif x % 5 == 0:
        print("buzz")
    else:
        print(x)
    x+= 1

File: loops/fizzbuzz.c
// Elizabeth Gutierrez, TEMPLATE
#include <stdio.h>

int main() {
    int x = 0;
    while (x < 51) {
        x = x + 1;
        if (x % 3 == 0 && x % 5 == 0) {
            printf("fizzbuzz\n");
        } else if (x % 3 == 0) {
            printf("fizz\n");
        } else if (x % 5 == 0) {
            printf("buzz\n");
        } else {
            printf("%d\n", x);
        }
    }
    return 0;
}

